### ISSUE TITLE
ValueError: 1 is not a valid UnitTypeId

### ISSUE THREAD

**Issue by njustesen** [Dentosal/python-sc2#5]
This looks great!
But when I run any of the examples I get:
```
Original exception was:
Traceback (most recent call last):
  File "/home/noju/Documents/python-sc2/examples/cannon_rush.py", line 60, in <module>
    main()
  File "/home/noju/Documents/python-sc2/examples/cannon_rush.py", line 57, in main
    ], realtime=False)
  File "/home/noju/Documents/python-sc2/sc2/main.py", line 105, in run_game
    _host_game(map_settings, players, **kwargs)
  File "/usr/lib/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "/home/noju/Documents/python-sc2/sc2/main.py", line 79, in _host_game
    result = await _play_game(players[0], client, realtime, portconfig)
  File "/home/noju/Documents/python-sc2/sc2/main.py", line 65, in _play_game
    return await _play_game_ai(client, player_id, player.ai, realtime)
  File "/home/noju/Documents/python-sc2/sc2/main.py", line 50, in _play_game_ai
    await ai.on_step(gs, iteration)
  File "/home/noju/Documents/python-sc2/examples/cannon_rush.py", line 21, in on_step
    if self.can_afford(PROBE):
  File "/home/noju/Documents/python-sc2/sc2/bot_ai.py", line 89, in can_afford
    cost = self._game_data.calculate_ability_cost(unit.creation_ability)
  File "/home/noju/Documents/python-sc2/sc2/game_data.py", line 28, in calculate_ability_cost
    if unit.id == ZERGLING:
  File "/home/noju/Documents/python-sc2/sc2/game_data.py", line 68, in id
    return UnitTypeId(self._proto.unit_id)
  File "/usr/lib/python3.6/enum.py", line 291, in __call__
    return cls.__new__(cls, value)
  File "/usr/lib/python3.6/enum.py", line 533, in __new__
    return cls._missing_(value)
  File "/usr/lib/python3.6/enum.py", line 546, in _missing_
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: 1 is not a valid UnitTypeId
```
Also, do I need a specific version of pysc2?

**Comment by Dentosal** [comment]
You are probably using an older version of `python-sc2`, since pypi repository was not updated for a while. I just published a new version (v0.4.0, 35a08231), and it should at least fix this issue.

**Comment by njustesen** [comment]
So I uninstalled sc2 and ran:
`sudo pip3 install sc2==0.4.0`
I get the same error.

Shouldn't I be able to run the examples when I clone the project without installing sc2? That also gives me the same error.

**Comment by njustesen** [comment]
Hi again,

I am hosting a StarCraft AI Workshop tomorrow for 100 people. If I can get this to run I will include it in my presentation. Any help is appreciated :)

Thanks

**Comment by Dentosal** [comment]
Hi!

I'm currently trying to reproduce the issue, as a clean install works well for me. However, I have not done a linux installation myself, as I'm developing on macOS.

Could you please include the following details:

* Operating system (Distro + version)
* StarCraft 2 version
* Output of `pip freeze | grep 'sc\?2'`

**Comment by njustesen** [comment]
Thanks,

I am on my other Ubuntu machine now but I get the same error.

Ubuntu 16.04
StarCraft II version 3.16.1 (I get another error with version 4.0.2).
```
$ pip freeze | grep 'sc\?2'
s2clientprotocol==4.1.3.61021.0
sc2==0.4.2
```

**Comment by Dentosal** [comment]
It will take me a while to install ubuntu, but I will hopefully be able to look at it later today.

**Comment by Dentosal** [comment]
I installed version 4.0.2, and indeed path resolution had a bug on Linux (fixed in 22ed1757272774c6636405c28fb0e9d8cba64283). It now run on my machine (Ubuntu 14.04, Python 3.6.4, StarCraft II 4.0.2).

**Comment by Dentosal** [comment]
Closing as fixed. Please reopen if the issue is still present.
